playbooks/templates/playbook.yml

```diff
@@ -0,0 +1,51 @@
+# Name your playbook, this is printed in STDOUT
+- name: My Cool Playbook
+  # Define your target hosts, this typically doesn't change
+  hosts: webportals
+  # Define the user that is executing the commands, this typically doesn't
+  # change
+  remote_user: "{{ webportal_user }}"
+  # Tell ansible if it should gather information about the hosts. This is
+  # currently only used in setting up the servers.
+  gather_facts: False
+
+  # Limit concurrency or the playbook if needed
+  serial: 1
+
+  # Execute playbook on hosts in parallel as fast as possible (do not wait for other hosts)
+  # strategy: free
+
+  # Stop on first error, do not execute on the next host
+  any_errors_fatal: True
+
+  # Playbook specific vars
+  vars:
+    max_hosts: "{{ groups['webportals'] | length - 1 }}"
+
+  # Define the tasks for your playbook
+  tasks:
+    - name: Hello World
+      ansible.builtin.command: echo "Hello World"
+
+    # Example of calling a refactored task
+    #
+    # Check '--limit' is used
+    - name: Fail if you are targeting all dev and prod webportals
+      include_tasks: tasks/host-limit-check.yml
+
+    # Example of task with additional vars
+    #
+    # Update log status to 'tested'
+    - name: Update log status to 'tested'
+      include_tasks: tasks/portal-logs-update-status.yml
```

Comment:
If this playbook/task is run against a portal host, it might set portal status file incorrectly, this can spoil possible rollback.

Reply:
The idea is that this is just an example.  I'm not suggesting that all playbooks would use these tasks.

Reply:
I understand the idea, but we should use tasks that do not interfere with our fleet when somebody tries to run the template playbook(s).